apps/chat/src/utils/app/overlay.ts

```diff
@@ -21,6 +21,7 @@ export function sendPMResponse(
   type: OverlayRequests,
   requestParams: PostMessageRequestParams,
 ) {
+  if (typeof window === 'undefined') return;
```

Comment:
can we move all `window`-related code in separate file and mark by `use client`?

Reply:
You a right, it's NextJS feature.
However, it doesn't work in this case.
For now this is only this solution works.